extensions/amp-addthis/0.1/addthis-utils/meta.js

```diff
@@ -31,3 +31,15 @@ export const getDetailsForMeta = meta => {
     content,
   };
 };
+
+export const getOgImage = doc => {
+  const ogImages = doc.head.querySelectorAll('meta[property="og:image"]');
```

Comment:
since you only care about the first one, please use `querySelector` instead of `querySelectorAll`

Reply:
Thanks @aghassemi. I just added a new commit that uses `querySelector` instead.